demisto_sdk/commands/lint/linter.py

```diff
@@ -482,7 +497,19 @@ def _setup_dev_files(self, py_num):
                 common_server_path = get_common_server_path(self.configuration.env_dir)
                 shutil.copy(common_server_path, self.project_dir)
         except Exception as e:
-            print_v('Could not copy demistomock and CommonServer files: {}'.format(str(e)), self.log_verbose)
+            print_v('Could not copy demistomock and CommonServer files: {}'.format(str(e)))
+
+    def _setup_dev_files_pwsh(self):
+        # copy common server
+        try:
+            shutil.copy(self.configuration.env_dir + '/Tests/demistomock/demistomock.ps1', self.project_dir)
+            if "/Scripts/CommonServerPowerShell" not in self.project_dir:
+                # Otherwise we already have the CommonServerPowerShell.py file
+                shutil.copy(self.configuration.env_dir + '/Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1',
+                            self.project_dir)
+        except Exception as e:
```

Comment:
this part should be changed in order to support base pack that should be merged today.
see:
https://github.com/demisto/demisto-sdk/pull/243/files

Reply:
For now let's leave the powershell common out from the base pack